static/index.js

```diff
@@ -6,6 +6,42 @@ if (location.protocol === 'https:') {
 const start = { x: 0, y: 0 };
 const appThemeLocalKey = "darkModeLocal";
 
+
+
+function map_actions(el){
+    if(['touchpad', 'vscrollpad', 'hscrollpad'].includes(el)){
+            return el.substring(0, el.length - 3);
+    }
+    else{
+        return el;
+    }
+}
+
+
+function getMetadata() {
+  let result = {}
+  const xhr = new XMLHttpRequest();
+  xhr.onreadystatechange = function() {
+        if (this.readyState == 4 && this.status == 200) {
+            result = JSON.parse(this.responseText);
+            let actions = ['toggle', 'mute', 'volume_down', 'volume_up', 'seek_left',
+                'seek_right', 'touchpad', 'vscrollpad', 'hscrollpad'];
+
+            let cleanActions = actions.map(map_actions)
+            for(var i=0; i<actions.length; i++) {
+                if (result["experimental-features"][cleanActions[i]] == false){
+                    document.getElementById(actions[i]).style.display = 'none';
+                    //only if hscroll is hidden - might not be worth trying to generalise?
```

Comment:
```suggestion
                    //only if hscroll is hidden - might not be worth trying to generalise?
```
I also think the same, can we instead maintain a `const` array of experimental features only (which is `hscroll` for now), and render only those. We won't have to go over all actions in that case.